sandbox/cgroup/cgroup.go

```diff
@@ -301,47 +301,61 @@ func MockPathOfProcPidCgroup(fn func(int) string) func() {
 // cgroups are not mounted.
 func ProcessPathInTrackingCgroup(pid int) (string, error) {
 	fname := pathOfProcPidCgroup(pid)
-	entries, err := loadProcCgroup(fname)
-	if err != nil {
-		return "", err
-	}
-	for _, e := range entries {
+	entry, err := scanProcCgroupFile(fname, func(e *procInfoEntry) bool {
```

Comment:
I think we could skip parsing the line too, but this will do.

It would be nice to add example of the lines we're looking for:
```
// cgroup entries we're looking for:
// 1:name=systemd:/user.slice/user-1000.slice/user@1000.service/tmux.slice/tmux@default.service
// 0::/user.slice/user-1000.slice/user@1000.service/tmux.slice/tmux@default.service

Reply:
I think the cost of parsing that over entirely over makeshift parsing is not worth at this time.

I will do a pass and add the comment.

Reply:
Done